src/hotspot/share/runtime/synchronizer.cpp

```diff
@@ -249,10 +251,11 @@ static MonitorList _in_use_list;
 // of the thread count derived ceiling because we have used more
 // ObjectMonitors than the estimated average.
 //
-// Start the ceiling with the estimate for one thread.
+// Start the ceiling with the estimate for one thread in initialize()
+// which is called after cmd line options are processed.
 // This is a 'jint' because the range of AvgMonitorsPerThreadEstimate
 // is 0..max_jint:
```

Comment:
Do we need this comment now?

Reply:
Yes. The range for AvgMonitorsPerThreadEstimate is still 0..max_jint.

Reply:
what would happen if this option was set to max_jint?  Is it a reasonable max?